msbuild-sonarqube-runner/msbuild-sonarqube-runner.nuspec

```diff
@@ -3,13 +3,15 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
   <metadata>
     <id>msbuild-sonarqube-runner</id>
-    <title>SonarQube Scanner for MSBuild</title>
-    <version>1.1.0.20160205</version>
+    <title>SonarQube Runner for MSBuild</title>
```

Comment:
Although the executable and the github repo is both uses "runner", the product name is "SonarQube Scanner for MSBuild", so we should keep that.